libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -59,12 +59,12 @@ interface MapboxNativeNavigator {
      * and verify that the user is still on the route. This method also determines
      * if an instruction needs to be called out for the user.
      *
-     * @param date the point in time to receive the status for.
+     * @param nanoTime the point in time to receive the status for. Unit is nano seconds
      *
      * @return the last [TripStatus] as a result of fixed location updates. If the timestamp
      * is earlier than a previous call, the last status will be returned. The function does not support re-winding time.
      */
-    suspend fun getStatus(date: Date): TripStatus
+    suspend fun getStatus(nanoTime: Long): TripStatus
```

Comment:
Why isn't the offset created inside MapboxNativeNavigator?

```suggestion
    suspend fun getStatus(navigatorPredictionMillis: Long): TripStatus
```

The time basis is smelling like an implementation detail of the navigator

Reply:
```suggestion
    suspend fun getStatus(navigatorPredictionNano: Long): TripStatus
```

> The time basis is smelling like an implementation detail of the navigator

I think it's a theme of another issue

Reply:
@kmadsen https://github.com/mapbox/mapbox-navigation-android/issues/3464